apps/omg_eth/test/omg_eth/root_chain_test.exs

```diff
@@ -63,8 +65,13 @@ defmodule OMG.Eth.RootChainTest do
 
   defp deposit_then_start_exit(owner, amount, currency) do
     owner = Encoding.from_hex(owner)
-    {:ok, deposit} = ExPlasma.Transaction.Deposit.new(owner: owner, currency: currency, amount: amount)
-    rlp = ExPlasma.Transaction.encode(deposit)
+
+    deposit =
+      ExPlasma.payment_v1()
+      |> Builder.new()
+      |> Builder.add_output(output_guard: owner, token: currency, amount: amount)
```

Comment:
Could we standardize around the key names instead? Moving towards `plasma-contracts` naming for these fields.

```suggestion
      |> Builder.add_output(output_guard: output_guard, token: token, amount: amount)
```